src/containers/tutor-profile/profile-info/ProfileContainerMobile.jsx

```diff
@@ -1,35 +1,41 @@
 import Box from '@mui/material/Box'
 import SchoolIcon from '@mui/icons-material/School'
 import DoneIcon from '@mui/icons-material/Done'
+import Avatar from '@mui/material/Avatar'
 
 import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
 import AppChipList from '~/components/app-chips-list/AppChipList'
 import ProfileDoneItemsList from '~/components/icon-with-text-list/ProfileDoneItemsList'
 
-import img from '~/assets/img/tutor-profile-page/avatar.png'
 import { styles } from '~/containers/tutor-profile/profile-info/ProfileInfo.styles'
 
 const ProfileContainerMobile = ({
   actionIcon,
   accInfo,
   buttonGroup,
   defaultQuantity,
-  subjectChips,
-  doneItems
+  doneItems,
+  userData
 }) => {
+  const subjectData = userData.mainSubjects.tutor.map((item) => item.name)
+
   return (
     <Box sx={styles.container}>
-      <Box sx={{ display: 'flex', gap: '10px' }}>
-        <Box sx={{ flex: 1 }}>
-          <Box component='img' src={img} sx={styles.img} />
+      <Box sx={styles.wrapperForPhoto}>
+        <Box sx={styles.avatarContainerMobile}>
+          <Avatar
+            src={
+              userData.photo &&
+              `${import.meta.env.VITE_APP_IMG_USER_URL}${userData.photo}`
+            }
+            sx={styles.img}
+          ></Avatar>
```

Comment:
closing tag is redundant if u don't pass children

Reply:
Fixed.